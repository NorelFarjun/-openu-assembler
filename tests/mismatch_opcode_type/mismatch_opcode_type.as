lea r3, r3
lea r3, #r3
lea r3, #1
clr #0
jmp r3
