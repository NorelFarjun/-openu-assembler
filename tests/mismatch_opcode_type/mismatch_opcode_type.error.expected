0: 'r3' is illegal as operand number 1 for lea
1: error with '#r3': Incorrect immediate value - not a number
2: '#1' is illegal as operand number 2 for lea
3: '#0' is illegal as operand number 1 for clr
4: 'r3' is illegal as operand number 1 for jmp
Bad input file - not outputting
